data/mihoyo

```diff
@@ -0,0 +1,5 @@
+# 米忽悠
+domain:mihoyo.com
+
+# 崩坏3
+domain:bh3.com
```

Comment:
Normally, `domain:` is omitted.

Reply:
I thought writing `domain:` will be more precise and more efficient when it comes to string matching.
Also I don't want to match something like `www.bh3.com.fake.example.com`.

Reply:
Resolved.

Reply:
> I thought writing `domain:` will be more precise and more efficient when it comes to string matching.
> Also I don't want to match something like `www.bh3.com.fake.example.com`.

https://github.com/v2ray/domain-list-community/blob/2daf08f27670f3d985e72618ce9081b21ad0fbf3/main.go#L84-L88

There's no difference whether you add `domain:`.